workers/data_refinery_workers/processors/smashing_utils.py

```diff
@@ -509,52 +511,33 @@ def _test_qn(merged_matrix):
     Returns a list of tuples with the results of the KN test (statistic, pvalue) """
     # Verify this QN, related:
     # https://github.com/AlexsLemonade/refinebio/issues/599#issuecomment-422132009
-    data_matrix = rlang("data.matrix")
-    as_numeric = rlang("as.numeric")
-    set_seed = rlang("set.seed")
-    combn = rlang("combn")
-    ncol = rlang("ncol")
-    ks_test = rlang("ks.test")
-    which = rlang("which")
-
-    merged_R_matrix = data_matrix(merged_matrix)
-
-    set_seed(123)
-
-    n = ncol(merged_R_matrix)[0]
-    m = 2
 
+    random.seed(123)
+    num_columns = merged_matrix.shape[1]
     # Not enough columns to perform KS test - either bad smash or single sample smash.
-    if n < m:
+    if num_columns < 2:
         return None
 
-    # This wont work with larger matricies
-    # https://github.com/AlexsLemonade/refinebio/issues/1860
-    ncolumns = ncol(merged_R_matrix)
```

Comment:
Do we still have this problem, such that we need the logic below, now that we're not passing things to R? We probably don't know _yet_ do we?

Reply:
I don't think we do, but I would be surprised if we did.